src/styles/utils/_extends.scss

```diff
@@ -0,0 +1,23 @@
+%animation-settings {
```

Comment:
It's recommended to use a more descriptive name for the placeholder, for example: %stopwatch-animation-settings